eventsourcing/domain/model/entity.py

```diff
@@ -86,7 +87,7 @@ def originator_topic(self):
             """
             return self.__dict__['originator_topic']
 
-        def __mutate__(self, entity_class=None):
+        def __mutate__(self, entity_class: Optional[Type[T]]) -> T:
```

Comment:
I think `=None` got dropped here. Further down you converted the same thing to this: `def __mutate__(self, entity_class: Optional[Type[T]] = None):`